site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "What's new in Svelte: August 2022"
+description: "Changes to SvelteKit's `load` before 1.0, better Vite configurations, plus some new HTML and CSS properties!"
```

Comment:
I'm not that familiar with the new html/css attributes, so am assuming they're not the biggest thing this month, but @dummdidumm can correct me on that if I'm wrong. Could probably mention Vite 3 instead 
```suggestion
description: "Changes to SvelteKit's `load` before 1.0 plus support for Vite 3 and `vite.config.js`!"
```